k8s/demo/cluster-00-overlay/kustomization.yaml

```diff
@@ -30,6 +30,7 @@ bases:
 
 patchesStrategicMerge:
   #flux patches
+  - ../../../namespaces/ccd/message-publisher/demo.yaml
```

Comment:
```suggestion
```
Since Demo is only one cluster we already have the generic demo.yaml applied to the common-overlay (see [here](https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/common-overlay/ccd/kustomization.yaml#L9)), so no need to update this file